src/userservice/logging.conf

```diff
@@ -28,6 +28,6 @@ formatter=generic
 args=(sys.stdout, )
 
 [formatter_generic]
-format=%(asctime)s | [%(levelname)s] | %(funcName)s | %(message)s
+format={"timestamp": "%(asctime)s", "message": "%(funcName)s | %(message)s", "severity": "%(levelname)s"}
```

Comment:
Quick question about the message: could it possibly break the JSON (i.e., do we have to worry about escaping "s in the message)?

Reply:
Interesting question. It doesn't break the entire logging per se, but it will break _those_ specific lines with quotes or new-line for example.

![image](https://user-images.githubusercontent.com/3271352/127537312-f117dc28-0809-46c2-a310-32f8bcb8e39c.png)

Reply:
I'm still messing with it to see if I can avoid this problem.

Reply:
I tried digging around some for e.g. Python and this is possible (escaping / removing certain characters like new-lines or quotes, for example) but it requires creating a custom handler typically.

Currently, none of the logs seem to use quotation or new-lines, so this problem doesn't arise.

With that in mind as well as thinking about the purpose of Bank of Anthos, does it make sense to add those lines of code which may not be related to Bank of Anthos and its purpose and add noise into the code? (This is more of an open question than a rhetoric, I don't know this answer to this.)

Logging docs + questions around the same topic:
https://docs.python.org/3/library/logging.html
https://stackoverflow.com/a/6692653
https://stackoverflow.com/a/13373364

Reply:
Does the issue arise if the message has single quotes as well? Or is it only a problem is it's a double quote?

Reply:
Double quote only. Single quotes are fine!
And any of those hypothetical "log breaking" logs wouldn't break the logs inside the Pod, nor does it break any _other_ log entries in Cloud Logging. Rather, it'll just break the _one_ log that has a special character (either `\n` or `"`, because those break the JSON format for that _one_ log entry).